src/App/Application.cpp

```diff
@@ -2525,32 +2501,32 @@ void processProgramOptions(const variables_map& vm, std::map<std::string,std::st
         mConfig["ExitTests"] = vm.count("run-open") == 0  ? "yes" : "no";
     }
 
-    if (vm.count("single-instance")) {
+    if (vm.count("single-instance") != 0U) {
         mConfig["SingleInstance"] = "1";
     }
 
-    if (vm.count("dump-config")) {
+    if (vm.count("dump-config") != 0U) {
         std::stringstream str;
         for (const auto & it : mConfig) {
-            str << it.first << "=" << it.second << std::endl;
+            str << it.first << "=" << it.second << '\n';
         }
-        throw Base::ProgramInformation(str.str());
+        throw ProgramInformation(str.str());
     }
 
-    if (vm.count("get-config")) {
-        std::string configKey = vm["get-config"].as<string>();
+    if (vm.count("get-config") != 0U) {
+        auto  configKey = vm["get-config"].as<string>();

```

Comment:
```suggestion
        auto configKey = vm["get-config"].as<string>();

```

Reply:
We try to follow the good old _The Boy Scout Rule: Leave your code better than you found it_.
In this case (like the alignment issue), you are leaving the code in a worse state (only marginally, but still).

Also, since Werner wants to go through and check all changes manually, I would like to reduce the mountain of work he's doing, not add to it.

Reply:
> Are you suggesting fixing A THOUSAND warnings should be held up by a few space characters?

This is only true if you aren't willing to do some minor changes based on feedback that you get.